bindings/python/pinocchio/robot_wrapper.py

```diff
@@ -175,6 +175,40 @@ def updateGeometryPlacements(self, q=None, visual=False):
     def framesForwardKinematics(self, q):
         pin.framesForwardKinematics(self.model, self.data, q)
 
+    def reduceModel(self,
```

Comment:
Maybe, it would be nice to just return a new robot, in order to keep the current one alive.

Reply:
I don't really know all of the use cases but it wouldn't seem naturally coherent with the rest of the API to me (not that this `reduceModel` does though). This would be the only method to return the object right?

Would it maybe make sense to have a static function that takes a robot and returns a new reduced one? Sort of like `pin.RobotWrapper.BuildFromURDF`.

Reply:
But it means that you will need to copy twice everything if you want to keep the original robot.
I would suggest rather renaming this method `buildReducedRobot` which returns a new robot, as done for `buildReducedModel`.

Reply:
> But it means that you will need to copy twice everything if you want to keep the original robot.
> I would suggest rather renaming this method `buildReducedRobot` which returns a new robot, as done for `buildReducedModel`.

I think I might be misunderstanding something: In order to return a new robot I would need to copy everything anyway right?

To be clear:
```python
reduced_robot = robot.buildReducedRobot(...)
```

In this case `reduced_robot` would need to have copies of every data structure in `robot`

Reply:
Yes, but you keep the orginal model for futur use

Reply:
I've added the change.